workflows/real_time_ai_surgical_video_processing/README.md

```diff
@@ -0,0 +1,130 @@
+# Real-Time AI Surgical Video Processing Workflow
+
+## Overview
+
+In this workflow, we demonstrate a comprehensive real-time AI pipeline for surgical video processing that includes:
+
+1. Out-of-body detection to determine if the endoscope is inside or outside the patient
+2. Dynamic flow condition based on out-of-body detection results:
+   - Deidentification (pixelation) when outside the body
+   - Multi-AI processing when inside the body
+3. Multi-AI processing with:
+   - SSD detection for surgical tool detection
+   - MONAI segmentation for endoscopic tool segmentation
+
+![Sample Endoscopy Image](images/RAISVP-sample-image.png)
+
+Fig. 1 Endoscopy (laparoscopy) image from a cholecystectomy (gallbladder removal surgery) showing tool detection and segmentation results from two concurrently executed AI models.
+Image courtesy of Research Group Camma, IHU Strasbourg and the University of Strasbourg ([NGC Resource](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/clara-holoscan/resources/holoscan_endoscopy_sample_data))
+
+Please refer to the README under [./app_dev_process](./app_dev_process/README.md) to see the process of developing the applications.
+
+The application graph looks like:
+![RAISVP-workflow](./images/RAISVP-dynamic-workflow.png)
+
+## Models
+
+This workflow utilizes three AI models:
+
+- [Out-of-body detection model](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/clara-holoscan/resources/endoscopy_out_of_body_detection): `out_of_body_detection.onnx`
+- [SSD model from NGC](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/clara-holoscan/resources/ssd_surgical_tool_detection_model) with additional NMS op: `epoch24_nms.onnx`
+- [MONAI tool segmentation model from NGC](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/clara-holoscan/resources/monai_endoscopic_tool_segmentation_model): `model_endoscopic_tool_seg_sanitized_nhwc_in_nchw_out.onnx`
+
+## Data
+
+[📦️ (NGC) Sample App Data for AI-based Endoscopy Tool Tracking](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/clara-holoscan/resources/holoscan_endoscopy_sample_data)
+
+[📦️ (NGC) Sample App Data for Out-of-Body Detection](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/clara-holoscan/resources/holoscan_endoscopy_sample_data)
+
+## Requirements
+
+Ensure you have installed the Holoscan SDK via one of the methods specified in [the SDK user guide](https://docs.nvidia.com/holoscan/sdk-user-guide/sdk_installation.html#development-software-stack).
+
+The directory specified by `--data` at app runtime is assumed to contain three subdirectories, corresponding to the NGC resources specified in [Model](#models) and [Data](#data): `endoscopy`, `endoscopy_out_of_body_detection`, `monai_tool_seg_model` and `ssd_model`. These resources will be automatically downloaded to the holohub data directory when building the application.
+
+## Building the application
+
+The repo level build command
+
+```sh
+./run build real_time_ai_surgical_video_processing
```

Comment:
We should recommend the build_and_run command instead

Reply:
I included both.